src/Compilers/Core/Portable/Operations/Statement.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    internal partial class VariableDeclaration : IVariableDeclaration
+    {
+        public VariableDeclaration(ILocalSymbol variable, IOperation initialValue, SyntaxNode syntax) :
+            this(variable,
+                initialValue,
+                variable == null || (initialValue != null && initialValue.IsInvalid),
```

Comment:
if you are talking about variable == null .. and stuff, in last design meeting, people were talking about removing IsInvalid parameter all together, so I will leave it for now.